delta/imagery/imagery_dataset.py

```diff
@@ -22,30 +22,40 @@
 import math
 import random
 import sys
+import os
+import portalocker
 
 import tensorflow as tf
 
 from delta.config import config
 from delta.imagery import rectangle
 from delta.imagery.sources import loader
 
+#import numpy as np
+
 class ImageryDataset:
     """Create dataset with all files as described in the provided config file.
     """
 
-    def __init__(self, images, labels, chunk_size, output_size, chunk_stride=1):
+    def __init__(self, images, labels, chunk_size, output_size, chunk_stride=1,
+                 resume_mode=False, log_folder=None):
```

Comment:
resume_mode doesn't sound like it would be a boolean. different name?

Reply:
Suggestions?  This is true when the dataset should skip already processed images.

Reply:
skip_processed=False?